language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/AbstractCompletionProvider.java

```diff
@@ -302,6 +302,10 @@ private List<LSCompletionItem> getTypeItems(BallerinaCompletionContext context)
                 String label = completionItem.getLabel();
                 String moduleName = importDeclarationNode.prefix().isEmpty() ? moduleSymbol.id().moduleName()
```

Comment:
What if we did `importDeclarationNode.moduleName(importDeclarationNode.moduleName() - 1).text()` instead of splitting string?

Reply:
+1